src/main/java/api/domain/cart/items/AddItemToCart.java

```diff
@@ -0,0 +1,92 @@
+package api.domain.cart.items;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class AddItemToCart {
```

Comment:
This class is redundant, please use Skus.java, all these fields are already in there. Objects should match the API naming, we should avoid creating objects for every operation we need in tests.

Reply:
done